.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-frobenius_norm
+ac3973e96086a42232cd726626b8165cd13b6ee9
```

Comment:
You probably need the commit hash on the pt/xla master:
282567afcc41d2d9703d7002f6a435ec9fc51447

As pt/xla pin usually does't work with fork PR.

Reply:
I've updated.